app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Nominations::NominateInductionCoordinatorController < ApplicationController
+  def new
+    @token = params[:token]
+    @nomination_email = NominationEmail.find_by(token: @token)
+
+    if @nomination_email.nil?
+      redirect_to link_invalid_nominate_induction_coordinator_path
+    elsif @nomination_email.expired?
+      redirect_to link_expired_nominate_induction_coordinator_path(school_id: @nomination_email.school_id)
+    elsif @nomination_email.school.fully_registered?
+      redirect_to already_nominated_request_nomination_invite_path
+    else
+      load_nominate_induction_tutor_form
+      @nominate_induction_tutor_form.token = @token
+    end
+  end
+
+  def create
+    load_nominate_induction_tutor_form
+
+    render :new and return unless @nominate_induction_tutor_form.valid?
+
+    @nominate_induction_tutor_form.save!
+    redirect_to nominate_school_lead_success_nominate_induction_coordinator_path
```

Comment:
Can't we just call this success_nominate_induction_coordinator_path ?

Reply:
There's already a `nominations/success` from requesting a nomination invite email.